pkgs/tools/misc/ncdu/default.nix

```diff
@@ -1,16 +1,30 @@
-{ lib, stdenv, fetchurl, ncurses }:
+{ lib, stdenv, fetchurl, fetchpatch, zig, ncurses }:
 
 stdenv.mkDerivation rec {
   pname = "ncdu";
-  version = "1.16";
+  version = "2.0";
 
   src = fetchurl {
     url = "https://dev.yorhel.nl/download/${pname}-${version}.tar.gz";
-    sha256 = "1m0gk09jaz114piidiw8fkg0id5l6nhz1cg5nlaf1yl3l595g49b";
+    sha256 = "sha256-Zs2mgEdnsukbeM/cqCX5/a9qCkxuQAYloBrVWVQYR8w=";
   };
 
+  patches = [
+    ./c-import-order.patch # https://code.blicky.net/yorhel/ncdu/issues/183
+  ];
+
+  XDG_CACHE_HOME="Cache"; # FIXME This should be set in stdenv
```

Comment:
If ``XDG_CACHE_HOME`` is unset it should fallback to ``$HOME/.cache``. So creating ``$HOME`` should be enough.

Reply:
Is there an idiomatic way of doing that?

Reply:
That should be done at the program level otherwise the implementation is wrong.

Reply:
We set $HOME to homeless-shelter in Nixpkgs though, don't we?